cdap-ui/app/cdap/components/Home/HomeActions/ActionCard.tsx

```diff
@@ -53,20 +53,26 @@ const styles = (theme): StyleRules => {
       color: theme.palette.grey[100],
       lineHeight: '1.2',
     },
-    footer: {
+    divider: {
       margin: '0 10px',
-      borderTop: `1px solid ${theme.palette.grey[500]}`,
-      padding: '10px 30px 18px',
+      borderColor: theme.palette.grey[500],
+    },
+    footer: {
       display: 'flex',
       justifyContent: 'center',
+      height: '48px',
     },
     link: {
-      width: '50%',
-      '&:last-child:not(:only-child)': {
-        textAlign: 'right',
-      },
-      '&:only-child': {
-        textAlign: 'center',
+      minWidth: '50%',
+      display: 'flex',
+      justifyContent: 'center',
+      alignItems: 'center',
+      flexGrow: 1,
+      color: 'var(--blue)',
```

Comment:
Possibly silly question: Why is the color defined using var() here (and on line 75) instead of getting them from theme.palette?

Reply:
it's being set for link color. For some reason, the link color is not one of the blue palette colors